### ISSUE TITLE
Various issues with v1.5 repository list

### ISSUE THREAD

**Issue by ianfore** [biocaddie/prototype_issues#160]
This is in 2.0 at datamedbeta.

On the list of repositories the icon, description and link for Vectorbase are OK - but the text says Dataverse.

**Comment by ianfore** [comment]
A number of repositories listed have the Dataverse description or, in one case, the Dryad description. I guess we may be waiting on some of the descriptions.

**Comment by RuilingLiu** [comment]
Hi Ian,

On the list of repositories, the title has been corrected.

**Comment by aegururaj** [comment]
@RuilingLiu , please fix the data repository descriptions for the various repos. A lot of them still have the Dataverse description.

**Comment by ianfore** [comment]
Thanks for the change!

**Comment by RuilingLiu** [comment]
The descriptions for these repositories have been fixed

**Comment by ianfore** [comment]
The Dataverse description needs attention. I am sure the description is accurate, likely copied from somewhere in Dataverse. However, this is describing Dataverse as a system for building repositories rather than indicating which Dataverse repository or repositories have been indexed. It's the content of a repository rather than the container that is important.

Morphobank description is a duplicate of BGI. 

The descriptions for the three Neurovault repositories are not helpful. If they are going to be listed as three different repositories we should at least provide descriptions that help a user understand what each one is.

Uniprot still has the Dryad description.

**Comment by RuilingLiu** [comment]
The descriptions of Morphobank and Uniprot have been fixed.

We can only find description for [Neurovault](http://neurovault.org/). @jgrethe mentioned that neurovault-atlas, neurovault-cols and neurovault-nidm are three different types of data. Should we merge them as one repository named neurovault?

**Comment by jgrethe** [comment]
We can do either for Neurovault. The three different datasets are atlases, analysis results, and images

**Comment by ianfore** [comment]
So @jgrethe you could say it's all there in the title :-) But we should say more.

30 seconds on neurovault gives me a clue about something that communicates better what it's all about "produced by MRI and PET studies".
